en/docs/quick-start-guides/deploy-your-first-automation.md

```diff
@@ -0,0 +1,127 @@
+    The setup wizard will install the Ballerina distribution required for Ballerina Integrator in to `<USER_HOME>/.ballerina/ballerina-home` directory.
+    Press `Ctrl + Shift + P` on Windows and Linux, or `shift + ⌘ + P` on a Mac and type `Ballerina: Update Ballerina Integrator` to update the installed Ballerina distribution.
+
+## Step 5: Develop automation in VSCode
+1. Click on **Develop automation in VSCode** in the automation overview page. This will open up your project on VSCode. <!-- Todo This is not working ATM update this if we are going live without this -->
+2. Click on the Ballerina Integrator icon on the sidebar. <!-- Todo step 1-6 will be automatically done by the BI extension. But that is not working ATM. Remove those steps once we support that -->
+3. Click on the **Create New Integration** button.
+4. Enter the Integration Name as `HelloWorld`.
+5. Project directory should be automatically populated. If not, select project directory by clicking on the **Select Location** button.
+6. Click on the **Create Integration** button to create the integration project.
+   <a href="{{base_path}}/assets/img/get-started/create_integration.gif"><img src="{{base_path}}/assets/img/get-started/create_integration.gif" alt="Create Integration" width="70%"></a>
+7. In the design view, click on the **Add Construct** button.
+8. Select **Automation** from the Constructs menu.
+9. Click **Create** to create an automation. This will direct you to the automation diagram view.
+10. Click on the **+** button after the **Start** node to open the node panel.
+11. Select **Function Call** and select **println**.
+12. Click on **+ Add Another Value**, type `"Hello World"` and click on **Save**.
+13. Click on **Run** button at the lop right corner to run the automation. The automation will be compiled and run in the embedded Ballerina runtime.
+   <a href="{{base_path}}/assets/img/get-started/design_integration.gif"><img src="{{base_path}}/assets/img/get-started/design_integration.gif" alt="Design Integration" width="70%"></a>
+
+## Step 6: Push to Devant <!-- Todo Update this if this is supported by the BI extension itself -->
+1. Click on **Source Control** icon on the sidebar.
+2. Click on **+** button to stage all the changes.
+3. Add a appropriate commit message and commit.
+4. Click on **Sync Changes** to push the changes to remote.
+
+## Step 7: Schedule Automation
+1. Once you push the changes, the overview page of the Devant automation will automatically refresh and show you the **Latest Commit** and automatically build your automation showing the **Build Status**.
+
+    !!! note
+        The build process may take some time. You can track progress in the **Build History** pane. Once complete, the build status changes to **Success**.
+
+2. Once the **Build Status** shows as `Build completed`, click on **Schedule** button.
+3. Switch to **BY INTERVAL** tab.
+4. Select **Minute** from the dropdown.
```

Comment:
change to daily.

Reply:
Since this is a quick start guide, IMO, the user should see at least a couple of executions of the automation. Changing it to daily won't allow that.

Reply:
I am worries we are encouraging people to create a lot of unnecessary load to Choreo.

1) Lets get users to experience the Run Now (now called "Test") feature. this will give them confidence it will work. This should come before the schedule step.
2) Let them see some schedule is being created, and see the "Executes in .." message
With 1 and 2, I don't think they need to see the schedule being executed.

Reply:
Okay, that makes sense. will update